src/components/Text.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+export default function Text({ 
+  text = "",
+  italic = false, 
+  bold = false, 
+  underline = false, 
+  size = 'text-base', // uses tailwind utility class to set 1rem
+  color = 'text-black', 
```

Comment:
@RaisaBh Did we ever discuss how colours would be passed in? Using this design pattern, FE devs would have to know directly the attributing tailwind-class-name for the specific colour they want to target...

This system doesn't allow for HEX to be passed in as a colour, since the styling is all done through Tailwind in this component, not CSS

Reply:
Actually this applies to all styling props. We should talk about this.

Reply:
For now we can keep what we have but once we start to figure out how to incorporate stuff like themes things will have to be changed.